labelbox/data/annotation_types/annotation.py

```diff
@@ -1,6 +1,8 @@
 import abc
 from typing import Any, Dict, List, Optional, Union
 
+from labelbox.data.mixins import ConfidenceNotSupportedMixin, ConfidenceMixin
```

Comment:
`labelbox/data` contains both the models and utilities for working with those models. All models are defined under `labelbox/data/annotation_types`. The mixins should be moved to the models dir (annotation_types) since they will never be used anywhere else.